bundles/org.openhab.core/src/main/java/org/eclipse/smarthome/core/library/items/DateTimeItem.java

```diff
@@ -38,10 +40,14 @@
     private static List<Class<? extends Command>> acceptedCommandTypes = new ArrayList<>();
 
     static {
-        acceptedDataTypes.add((DateTimeType.class));

```

Comment:
I don't think the accepted datatypes should be changed - after all, it expects data time values and nothing else.
Note that `DateTimeType` has a constructor that also takes Strings, so this is probably rather the way you'd go for?

Reply:
Yes, I was not sure of my shot on this one. Thanks to point it.